Include/py_curses.h

```diff
@@ -64,6 +62,13 @@ typedef struct {
     char *encoding;
 } PyCursesWindowObject;
 
+typedef struct {
+    PyTypeObject *Window_Type;
+    int (*SetupTermCalled)(void);
+    int (*Initialised)(void);
+    int (*InitialisedColor)(void);
```

Comment:
"Initialised" (S) is the british spelling, "color" (no U) is the US spelling... In Python, we prefer the US spelling: "initialized" (Z):

```suggestion
    PyTypeObject *window_type;
    int (*setup_term_called)(void);
    int (*initialized)(void);
    int (*initialized_color)(void);
```

Reply:
Thanks, got it. Updated ;)